packages/cumulus-api-client/tests/test-CumulusApiClient.js

```diff
@@ -12,7 +14,7 @@ const CumulusAuthTokenError = require('../CumulusAuthTokenError');
 
 test.before(async (t) => {
   process.env.tableName = randomId('table');
-  dynamodb().createTable({
+  await dynamodb().createTable({
```

Comment:
You can use this here instead:

https://github.com/nasa/cumulus/blob/master/packages/aws-client/DynamoDb.js#L139

Reply:
💯  nice

Reply:
df015f064